backend/effects/builtin/conditional-effects/conditions/builtin/viewer-roles.js

```diff
@@ -44,21 +48,21 @@ module.exports = {
     },
     predicate: async (conditionSettings, trigger) => {
 
-        let { comparisonType, rightSideValue } = conditionSettings;
+        let { comparisonType, leftSideValue, rightSideValue } = conditionSettings;
+        let username = leftSideValue;
 
-        let username = trigger.metadata.username;
         if (username == null || username === "") {
-            return false;
+            username = trigger.metadata.username;
         }
```

Comment:
I believe this will cover people using the old version of the conditional. If leftSideValue is blank, then use the username of the person triggering the conditional.